domains/auth/Auth.API/Tests/Tests.csproj

```diff
@@ -11,6 +11,7 @@
 	  <PackageReference Include="AutoFixture" Version="4.17.0" />
 	  <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.11.0" />
 	  <PackageReference Include="Moq" Version="4.17.2" />
+	  <PackageReference Include="RichardSzalay.MockHttp" Version="6.0.0" />
```

Comment:
Why are we pulling in yet another dependency? Has anyone verified this http mocking tool is the industry standard to use?

Reply:
@ckr123 I have talked with @CodeReaper and I will try to do an implementation that is not using this package. I will fill you in on the details on Monday or you can help me find a nice solution.

Reply:
@ckr123 and @CodeReaper I have made an example using Moq for doing the same here: 5e9dccb5d37656fd60ebd61aa8a83f61118f3ee5

If we want to use a library for easy mocking of HttpClient, then RichardSzalay.MockHttp is the most popular/industry standard.

Reply:
As discussed we will leave it as it is right now and bring this up at the developer meeting